en/starter/settingup.md

```diff
@@ -24,7 +24,7 @@ redirect_from: "/starter/setting up.html"
 <hr>
 ## Ubuntu Linux
 
-Kitura is tested on Ubuntu 14.04 LTS, Ubuntu 15.10, and Ubuntu 16.04 LTS.
+Kitura is tested on Ubuntu 14.04 LTS.
```

Comment:
The work to add a Ubuntu 16.04 travis build is ready to go for Kitura and Kitura-net and is just waiting on https://github.com/IBM-Swift/Package-Builder/issues/104 because our branch for ubuntu 16.04 currently has a patch for that issue, which I'd like to remove when the Package-Builder issue has been closed.